src/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -94,13 +100,14 @@ export class WorkAllocationService {
    * @param ccdId The ID of the case to find tasks for.
    * @param eventId The ID of the event to find tasks for.
    */
-  public completeAppropriateTask(ccdId: string, eventId: string): Observable<any> {
-    const parameters: TaskSearchParameter[] = [{
+  public completeAppropriateTask(ccdId: string, eventId: string, jurisdiction: string, caseTypeId: string): Observable<any> {
```

Comment:
The change to this signature theoretically makes this a breaking change.

Reply:
Again, this is used internally within the CCD-UI toolkit. Not sure why you think it will be a breaking change. Please explain. Thanks

Reply:
Again, because it's a `public` method. I wanted to make sure it doesn't form part of the public api for the toolkit. If it _does_, it would constitute a breaking change. If it's _entirely_ internal, it would be fine.

Reply:
Its not being used anywhere. But, Theoretically I agree. let me increment the version to major version.

Reply:
2.79.0